packages/jsoo-react/jsoo-react.0.1/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings to ReactJS for js_of_ocaml, including JSX ppx"
+maintainer: ["Javier Chávarri <javier.chavarri@gmail.com>"]
+authors: ["Javier Chávarri <javier.chavarri@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/ml-in-barcelona/jsoo-react"
+bug-reports: "https://github.com/ml-in-barcelona/jsoo-react/issues"
+depends: [
+  "dune" {>= "2.7" & >= "2" & < "4"}
+  "ocaml" {>= "4.12.0" & < "5.0.0"}
+  "js_of_ocaml" {>= "4.0.0" & < "5.2.0"}
+  "gen_js_api" {>= "1.0.8" & < "1.2.0"}
```

Comment:
```suggestion
  "gen_js_api" {>= "1.0.8"}
```
Are you sure about this upper bound? We check if these are needed when the packages are release and we re-add them manually where needed if that is the case